peakina/readers/excel.py

```diff
@@ -0,0 +1,199 @@
+"""
+Module to add xml support
+"""
+import enum
+from io import StringIO
+from typing import Any, Generator, List, Optional, Tuple, Union
+
+import pandas as pd
+import xlrd
+from openpyxl import load_workbook
+from openpyxl.utils.exceptions import InvalidFileException
+
+from peakina.readers.common import PreviewArgs
+
+
+class EXCEL_TYPE(enum.Enum):
+    NEW = "new"
+    OLD = "old"
```

Comment:
Could you please add a docstring here explaining what is "old" and what is "new"? thanks!

Reply:
Reading the rest of the PR, i deduced that this is not necessarily related to the generation of the excel file, but the ability to open it with openpyxl or xlrd.

May I suggest:
- either to rename this to `WORKBOOK_TYPE` with `OPENPYXL` and `XLRD`
- or to remove the need for his enum and detect which class we have. For xlrd, the `wb` is an instance of [xlrd.book.Book](https://xlrd.readthedocs.io/en/latest/api.html#xlrd.book.Book), for openpyxl it's an instance of [openpyxl.workbook.Workbook](https://openpyxl.readthedocs.io/en/stable/api/openpyxl.reader.excel.html?highlight=load_workbook#openpyxl.reader.excel.load_workbook)

My preference goes with the second ^^

Reply:
True, it sound better !
👍🏾 Updated !